xdcs-frontend/src/app/view/task-creation/task-creation.component.less

```diff
@@ -0,0 +1,70 @@
+.container-fluid {
```

Comment:
do czego to jest wgl?

PS nie zapinaj się na klasy z bootstrapa tylko dodawaj swoje jak już

Reply:
zmieniłem i zapomniałem usunąć